### DISCUSSION TITLE
Write fast datatype for BOOL, DINT, and STRING

### DISCUSSION THREAD

**Discussion by sophiabrewing** [dmroeder/pylogix#254] [Q&A]
I saw the example files and on the write fast example they have specified a datatype = 196. If I am working with a BOOL, DINT, STRING what number should I use to specify the datatype o can I type in 'BOOL', 'DINT', and 'STRING'.

**Comment by dmroeder** [comment]
DINT = 196 (0xc4), STRING = 160 (0xa0), BOOL = 193 (0xc1)

https://github.com/dmroeder/pylogix/blob/master/pylogix/eip.py#L62-L84

Strings are odd because in Rockwell land, they are a STRUCT (UDT).  You will see a STRING type, this is implemented by Micro800
